ipyleaflet/leaflet.py

```diff
@@ -2031,6 +2032,38 @@ def remove_legend_element(self, key):
         self.send_state()
 
 
+class ColormapControl(WidgetControl):
+    """ColormapControl class, with WidgetControl as parent class.
+
+    A control which contains a colormap, to be used with Choropleth.
+
+    Attributes
+    ----------
+    caption : str, default 'caption'
+        The caption of the colormap.
+    colormap_choice : str, default 'linear.YlOrRd_04'
```

Comment:
In the `Choroplet` class, this attribute is called `colormap`, which I prefer.
Also, the type is not a string but the Branca object itself.